messari-cli/bin/env.ts

```diff
@@ -0,0 +1,9 @@
+const execSync = require('child_process').execSync
+
+// Change to the directory of this file
+process.chdir(__dirname)
+
+// Get the path to the head of the `Subgraphs` repo
+export const MESSARI_REPO_PATH = execSync('git rev-parse --show-toplevel')
```

Comment:
This assumes there will only be one local git clone of the repo on each system. It might be true 99% of the time, so it is probably fine. But if we ever make it an npm package we migth want to be able to set the `MESSARI_REPO_PATH` env var dinamically or something.

Don't need to change it now, but let's keep it in mind for potential updates. It might be worth documenting it too in the cli README so people are aware though.

Reply:
Yeah - I had considered this. I chose to do it this way, for now because it was easier. But on second thought, I might have to make this change soon, since I am not sure if the path will be set correctly in the build checker CI action.